docs/install-1.6.md

```diff
@@ -165,3 +169,38 @@ kubectl config set-context service-catalog --cluster=service-catalog
 Note: Your cloud provider may require firewall rules to allow your traffic get in.
 Please refer to the [Troubleshooting](./walkthrough-1.6.md#troubleshooting) 
 section of the walkthrough document for details.
+
+## Troubleshooting
+
+### Non-default image installation
+
+The service catalog installation will by default use images of the controller
+and apiserver built at the latest tagged version. Should you be using a
```

Comment:
quibble over what 'latest tagged version` means.

`most recent version with a git tag`? It's the `git tag` that drives it. 

Maybe a `for example, v0.0.16` would clear it up?

Reply:
Yeah, reading over that again, it seems ambiguous. I think `built from the latest versioned release (for example, v0.0.16)` instead should cover all the bases.